web/js/jquery.websocket.js

```diff
@@ -35,7 +35,12 @@
                     .bind('message', function(e) {
                         var m = JSON.parse(e.originalEvent.data);
                         var h = settings.events[m.type];
-                        if (h) h.call(this, m);
```

Comment:
Try catching these in jquery.jmri.js. That's where we care which types exist. The code in this file is deliberately unconcerned about the types.